packages/botonic-api/src/rest/routes/events.ts

```diff
@@ -82,20 +81,14 @@ router
       res.status(500).send({ error: e.message })
     }
   })
-  .put(checkSchema({ eventId: eventIdParamSchema }), async (req, res) => {
-    const errors = await validateBotonicEventData(req)
+  .put(async (req, res) => {
+    const errors = await validateBotonicEventData(req, false, true)
```

Comment:
named parameters better? it's hard to understand what this does without looking at the function signature.

Reply:
Agree, changed!